openedx/core/djangoapps/content/course_overviews/management/commands/tests/test_generate_course_overview.py

```diff
@@ -56,13 +56,12 @@ def test_generate_one(self):
         self._assert_courses_in_overview(self.course_key_1)
         self._assert_courses_not_in_overview(self.course_key_2)
 
-    @patch('openedx.core.djangoapps.content.course_overviews.management.commands.generate_course_overview.log')
-    def test_invalid_key(self, mock_log):
+    def test_invalid_key(self):
         """
         Test that invalid key errors are logged.
         """
-        self.command.handle('not/found', all=False)
-        self.assertTrue(mock_log.fatal.called)
+        with self.assertRaises(CommandError):
```

Comment:
It tests the `InvalidKeyError` case.
The "No courses specified" case should never be reached. So I'm removing those lines from the code.